project/src/main/webapp/navigation/header.component.js

```diff
@@ -0,0 +1,23 @@
+//Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// use modern JavaScript (ES5)
+"use strict"
+
+angular.module('navigation').component('header', {
+  template: `<h1>Voter Central</h1>`,
+  controller: function generateHeader() {
+    // Generate any dynamic info for header
+  }
+});
```

Comment:
nit. add new line at end of file

Reply:
Aw jeez. I really thought I caught all of these this time.